src/widget/woverview.cpp

```diff
@@ -342,6 +346,31 @@ void WOverview::onRateRatioChange(double v) {
     update();
 }
 
+void WOverview::onPassthroughChange(double v) {
+    m_bPassthroughEnabled = static_cast<bool>(v);
+
+    if (m_bPassthroughEnabled) {
+        if (m_pCurrentTrack != nullptr) {
+            disconnect(
+                    m_pCurrentTrack.get(),
+                    &Track::waveformSummaryUpdated,
+                    this,
+                    &WOverview::slotWaveformSummaryUpdated);
```

Comment:
I am a bit concerned that now two parts are messing with the waveformSummaryUpdated signal. 
It is hard to check from the code if all edge case are considered.  
How about early return in slotWaveformSummaryUpdated if m_bPassthroughEnabled?

Reply:
Ok, I'll look into that.

Reply:
Done.